src/cloud-api-adaptor/pkg/adaptor/proxy/proxy.go

```diff
@@ -50,32 +43,30 @@ type AgentProxy interface {
 }
 
 type agentProxy struct {
-	tlsConfig     *tlsutil.TLSConfig
-	caService     tlsutil.CAService
-	readyCh       chan struct{}
-	stopCh        chan struct{}
-	serverName    string
-	socketPath    string
-	criSocketPath string
-	pauseImage    string
-	proxyTimeout  time.Duration
-	criTimeout    time.Duration
-	stopOnce      sync.Once
+	tlsConfig    *tlsutil.TLSConfig
+	caService    tlsutil.CAService
+	readyCh      chan struct{}
+	stopCh       chan struct{}
+	serverName   string
+	socketPath   string
+	pauseImage   string
+	proxyTimeout time.Duration
+	criTimeout   time.Duration
```

Comment:
I don't think it needs `criTimeout` anymore as it was used on `initCriClient` (function removed) to initialize the cri client.

I failed to find a user parameter to set the cri timeout so I assume it always used the `defaultCriTimeout`.